File: 백준/Silver/13241. 최소공배수/최소공배수.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        long A = sc.nextLong();
        long B = sc.nextLong();
        long gcd = getGCD(Math.max(A, B), Math.min(A, B));
        
        System.out.println(A * B / gcd);
    }
    
    public static long getGCD(long A, long B) {
        while(B > 0) {
            long tmp = A;
            A = B;
            B = tmp % B;
        }
        return A;
    }
}

File: 백준/Silver/13241. 최소공배수/최소공배수.cs
using System;

class Q13241
{
    static void Main(string[] args)
    {
        string[] S = Console.ReadLine().Split();
        long A = long.Parse(S[0]);
        long B = long.Parse(S[1]);
        long gcd = getGCD(Math.Max(A, B), Math.Min(A, B));
        
        Console.WriteLine(A * B / gcd);
    }
    
    public static long getGCD(long A, long B)
    {
        while (B > 0)
        {
            long tmp = A;
            A = B;
            B = tmp % B;
        }
        
        return A;
    }
}